google/resource_compute_instance.go

```diff
@@ -280,6 +288,13 @@ func resourceComputeInstance() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"create_timeout": &schema.Schema{
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  4,
```

Comment:
I think we want to remove defaults. I'm not sure if they do anything, but let's not take the chance.